Refresh.GameServer/Database/RealmDatabaseContext.Relations.cs

```diff
@@ -101,6 +105,8 @@ public bool UnfavouriteUser(GameUser userToFavourite, GameUser userFavouriting)
         {
             this._realm.Remove(relation);
         });
+        
+        // this.CreateUserUnfavouriteEvent(userFavouriting, userToFavourite);
```

Comment:
broken code should be documented as such

Reply:
Just gonna strip it entirely - I'll add it back if it does ever get used